ui/raidboss/data/05-shb/alliance/the_copied_factory.js

```diff
@@ -103,29 +97,7 @@
       regexDe: Regexes.startsUsing({ id: '48CE', source: 'Befehlsmodell Mit Omnigelenk' }),
       regexFr: Regexes.startsUsing({ id: '48CE', source: 'Modèle Multiarticulé : Commandant' }),
       regexJa: Regexes.startsUsing({ id: '48CE', source: '多関節型：司令機' }),
-      alertText: function(data, matches) {
-        if (matches.target == data.me) {
-          return {
-            en: 'Tank Buster on YOU',
-            de: 'Tankbuster auf DIR',
-            fr: 'Tankbuster sur VOUS',
-            cn: '死刑点名',
-            ko: '탱버 대상자',
-          };
-        }
-      },
-      infoText: function(data, matches) {
-        if (matches.target == data.me || data.role != 'healer')
-          return;
```

Comment:
what are those 
```javascript
      infoText: function(data, matches) {
        if (matches.target == data.me || data.role != 'healer')
          return;
```
was for?
my brain cannot logically understand this.
if healer, does not send message?

Reply:
I think this is a bad copy and paste.  I am not sure either.